src/mouse_events.cpp

```diff
@@ -1630,6 +1670,69 @@ std::set<map_location> mouse_handler::get_adj_enemies(const map_location& loc, i
 	return res;
 }
 
+/**
+ * NOT NEEDED now, keeping for future work on ranged attacks UI
```

Comment:
This does not seem sensible. If you want to keep code around for future work then do that privately. No point in leaving in dead code.